web/js/diceware/random-integer.js

```diff
@@ -0,0 +1,36 @@
+export { getLocalIntegers };
+
+async function getLocalIntegers(count,min,max) {
+    count = count || 2;
+    min = min || 0;
+    max = max || 1E9;
+
+    try {
+        const crypto = window.crypto || window.msCrypto;
+    } catch (err) {
+        console.error('crypto support is disabled!');
+    }
+
+    var results = [];
+
+    // newer, more secure random crypto supported?
+    if (
+        (crypto && crypto.getRandomValues)
```

Comment:
Generally I agree with being as robust as possible, and this fallback would normally be quite reasonable.

However, given that we're going to be feature-detecting for the whole `crypto`, and if it's not there we're blocking users anyway -- because privacy/security are feature number 1 -- then I believe we can drop needing to fallback to `Math.random` and just assume that all users of the app will have `getRandomValues`.